Assets/_Scripts/Character/StatusAilment/StatusAilment.cs

```diff
@@ -2,10 +2,11 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public abstract class StatusAilment {
+public abstract class StatusAilment : ScriptableObject {
 	public string name;
 	public int duration;
 	public int level;
+	[Header("Phase to trigger Status Ailment")]
```

Comment:
Minor, but I don't think it's common to add a header for a variable description. Headers are to group variables into categories. I think Tooltips are what you want to use.

Reply:
good point